lib/ui/src/components/sidebar/treeview/treeview.tsx

```diff
@@ -49,6 +49,7 @@ const linked = (
   }: { onClick: Function; onKeyUp: Function; Link: ComponentType; prefix: string }
 ) => {
   const Linked = React.memo(p => (
+    // @ts-ignore
```

Comment:
For some reason, compiling TS doesn't quite ignore the `onKeyUp`, it references line 52 instead so I had to add a `// @ts-ignore` here. I'm not sure if this is different in TS3.7 vs TS3.6.